hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -261,6 +261,10 @@ private ContainerCommandResponseProto dispatchRequest(
         // If container does not exist, create one for WriteChunk and
         // PutSmallFile request
         responseProto = createContainer(msg);
+        metrics.incContainerOpsMetrics(Type.CreateContainer);
+        metrics.incContainerOpsLatencies(Type.CreateContainer,
+                System.currentTimeMillis() - startTime);
```

Comment:
Let's use Time.monotonicNow(). (Read the Javadoc for why this is recommended over System.currentTimeMillis())
Let's also update StartTime to use Time.monotonicNow() too.

Reply:
oh!! checking

Reply:
There's another call to `incContainerOpsLatencies`, for the real incoming command:

https://github.com/apache/ozone/blob/7e3a77508f450121ba10d3a9d5a6713684cbf031/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java#L320-L321

I think we should change to `Time.monotonicNow()` there, too.

Reply:
oh!! good catch, thanks Attila! updated